### PR TITLE
Add View Resource Page

### PR DESCRIPTION
**Pull request by lucindan**
-Using Montreal AI Ethics' Living Dictionary as an example
-Using same menu as "Admin page"
-Broken into description, details, and files

### CODE REVIEW

**Comment by sshh12** [comment]
FYI @Marthacz you can review this page when merged at /resources/example

File: src/App.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import Register from './views/Register';
 import Resources from './views/Resources';
 import AddResources from './views/AddResource';
 import Admin from './views/Admin';
+import ResourceInfo from './views/ResourceInfo';
```
Comment (sshh12) [inline src/App.js:12]: Can we make this "ViewResource", this will match our existing view naming,

File: src/views/ResourceInfo.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+}
+
+
+export default function ShowExample() {
+  return (
+    <ResourceInfo resource = {resourceData}/>
+  );
+}
+
+function ResourceInfo(props) {
```
Comment (sshh12) [inline src/views/ResourceInfo.js:133]: "ViewResource". Also is what should be exported (you can prob delete ShowExample and just define the resource example data inside the component).

File: src/views/ResourceInfo.js [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+                  key="3" 
+                  shape="round">
+                    Edit Resource
+                </Button>,
+              ]}
+            >
+              {props.resource.desc}
+            </PageHeader>,
+          </div>
+          <div ref={detailRef}>
```
Comment (sshh12) [inline src/views/ResourceInfo.js:168]: Can we add more padding to the left and right of this component. Right now it's flush with the sides but there should be a small gap.

**Review by sshh12 — CHANGES_REQUESTED** [review]
Looks good, some small tweaks.

Also there seems to be a lot of little formatting things. Make sure you have the `esbenp.prettier-vscode` vscode plugin and format these (this plugin will auto format whenever you save the file)

**Comment by Marthacz** [comment]
> FYI @Marthacz you can review this page when merged at /resources/example

Thanks! I'll review once merged.

**Comment by lucindan** [comment]
-Renamed to ViewResource
-Added padding

**Review by sshh12 — APPROVED** [review]

**Pull request merged**